internal/commands/loglevel/set.go

```diff
@@ -62,22 +62,24 @@ func updateDebuggingConfiguration(ctx context.Context, logSeverity libsveltosv1a
 		)
 	}
 
-	return updateLogLevelConfiguration(ctx, spec)
+	return updateLogLevelConfiguration(ctx, spec, namespace, clusterName)
 }
 
 // Set displays/changes log verbosity for a given component
 func Set(ctx context.Context, args []string) error {
 	doc := `Usage:
-  sveltosctl log-level set --component=<name> (--info|--debug|--verbose)
+  sveltosctl log-level set --component=<name> --namespace=<namespace> --cluster=<cluster-name> (--info|--debug|--verbose)
 Options:
   -h --help             Show this screen.
      --component=<name> Name of the component for which log severity is being set.
```

Comment:
nit: can you please make sure indentation for the comment is same for all options? it makes it easier to read when doing

sveltosctl low-level set --help